README.md

```diff
@@ -218,3 +218,17 @@ Starting with version 2.8, there is the `ide` flavor of the images, which comes
 Store your preferred password in this variable if you need to password protect the IDE environment.
 
 [See docs](https://docs.docksal.io/tools/ide/) for instructions on using Coder in Docksal.
+
+## Composer
+
+Versions 1.x and 2.x are apart of the container. While not all projects will work with Version 2.x quite yet 
```

Comment:
"apart" means to be separate. Do you mean "a part" of the container? If so, it can just read Versions 1.x and 2.x are part of the container.

Reply:
@shelane should have been "a part"